activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeStandaloneTaskTest.java

```diff
@@ -167,5 +170,110 @@ public void createStandaloneTaskFailWithEmptyName() {
 
         assertThat(tasks.getContent()).hasSize(0);
     }
+    
+    @Test
+    public void shouldThrowExceptionOnTaskSave_when_wrongVariableName() {
```

Comment:
"should_" for all methods

Reply:
Will be done

Reply:
you want to do it as part of this pr or not?

Reply:
Is changed